From 5242ce04f64ae599eec17f5c7afcf500a64418c7 Mon Sep 17 00:00:00 2001
From: "Wang, Liang" <liang.wang@intel.com>
Date: Fri, 28 Jan 2022 05:14:44 +0000
Subject: [PATCH] adb: Don't listen on default port for security reason.

We need to disable network adb by default on cloud usage.

Change-Id: Ic8c09475279ea4320832b2f881cd94e5efc2a4eh
Signed-off-by: Wang, Liang <liang.wang@intel.com>
---
 adb/daemon/main.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/adb/daemon/main.cpp b/adb/daemon/main.cpp
index 7a0f7ffcd..838c6c618 100644
--- a/adb/daemon/main.cpp
+++ b/adb/daemon/main.cpp
@@ -236,7 +236,8 @@ int adbd_main(int server_port) {
     // adbd_auth_init will spawn a thread, so we need to defer it until after selinux transitions.
     adbd_auth_init();
 
-    bool is_usb = false;
+    // Don't listen on default port to avoid security concerns on cloud usage
+    bool is_usb = true;
 
 #if defined(__ANDROID__)
     if (access(USB_FFS_ADB_EP0, F_OK) == 0) {
-- 
2.39.0

